RegistrationApp/RegistrationApp/ViewModels/SignUpViewModel.cs

```diff
@@ -79,19 +96,24 @@ private void OnCreateAccauntCommand()
         {
             System.Diagnostics.Debug.WriteLine("Create accaunt");
 
-            if (!Email.Validate() || !Password.Validate())
+            if (!Email.Validate() || !Password.Validate() || !ConfirmPassword.Validate())
             {
                 var errorMessage = string.Empty;
                 if (Email.ErrorsMessages.Any())
                 {
                     errorMessage = Email.ErrorsMessages.First();
                 }
-                else
+                else if (Password.ErrorsMessages.Any())
                 {
                     errorMessage = Password.ErrorsMessages.First();
                 }
+                else
+                {
+                    errorMessage = ConfirmPassword.ErrorsMessages.First();
```

Comment:
It is not a mistake
so the best practices use FirstOrDefault() it works without any exceptions

Reply:
@Adyvan FYI:
when you use `FirstOrDefault` it's mean that your object could be 'null'.
But in this situation you show that object can't be null.